frontend/src/_data/audios.js

```diff
@@ -0,0 +1,8 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+module.exports = async function () {
+  const audiosdata = require('../../../data/audios.json');
+  return audiosdata;
```

Comment:
No need keeping a local variable. Return the JSON right away.

Reply:
local variable removed